### PR TITLE
SRVCOM-1245: Update and improvements to eventing docs structure

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for OCP 4.6+

https://issues.redhat.com/browse/SRVCOM-1245

Direct preview links:

- https://deploy-preview-33465--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_eventing/serverless-using-brokers (Brokers)
- https://deploy-preview-33465--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_eventing/serverless-triggers.html (Triggers)
- https://deploy-preview-33465--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_eventing/serverless-event-delivery.html (Event delivery)
- https://deploy-preview-33465--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_eventing/serverless-kafka.html (Kafka)

QE reviewed by @mgencur

### CODE REVIEW

File: modules/apiserversource-yaml.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ This guide describes the steps required to create an API server source using YAM
 
 .Procedure
 
-. To create a service account, role, and role binding for the API server source, create a file named `authentication.yaml` and copy the following sample code into it:
+. Create a service account, role, and role binding for the API server source, by entering the following command:
```
Comment (abrennan89) [inline modules/apiserversource-yaml.adoc:17]:
```suggestion
. Create a service account, role, and role binding for the API server source:
```

File: modules/apiserversource-yaml.adoc [hunk 2]
```diff
@@ -90,17 +83,7 @@ spec:
...
-+
-[source,terminal]
-----
-$ oc apply -f k8s-events.yaml
-----
-
-. To check that the API server source is set up correctly, create a Knative service that dumps incoming messages to its log.
-+
-Copy the following sample YAML into a file named `service.yaml`:
+. To check that the API server source is set up correctly, create a Knative service that dumps incoming messages to its log, by entering the following command:
```
Comment (abrennan89) [inline modules/apiserversource-yaml.adoc:86]:
```suggestion
. To check that the API server source is set up correctly, create a Knative service that dumps incoming messages to its log:
```

File: modules/apiserversource-yaml.adoc [hunk 3]
```diff
@@ -115,15 +98,8 @@ spec:
...
-+
-After you have created the `service.yaml` file, apply it:
-+
-[source,terminal]
-----
-$ oc apply -f service.yaml
-----
 
-. To create a trigger from the `default` broker that filters events to the service created in the previous step, create a file named `trigger.yaml` and copy the following sample code into it:
+. Create a trigger from the `default` broker that filters events to the service created in the previous step, by entering the following command:
```
Comment (abrennan89) [inline modules/apiserversource-yaml.adoc:102]:
```suggestion
. Create a trigger from the `default` broker that filters events to the service created in the previous step:
```

File: modules/serverless-create-kafka-channel-yaml.adoc [hunk 4]
```diff
@@ -15,7 +15,7 @@ You can create a Kafka channel by using YAML to create the `KafkaChannel` object
 
 .Procedure
 
-. Create a YAML file and copy the following sample code into it:
+* Create a `KafkaChannel` object by entering the following command:
```
Comment (abrennan89) [inline modules/serverless-create-kafka-channel-yaml.adoc:18]:
```suggestion
* Create a `KafkaChannel` object:
```

File: modules/serverless-kafka-source-kn.adoc [hunk 5]
```diff
@@ -26,30 +27,35 @@ $ kn service create event-display \
 +
 [source,terminal]
 ----
-$ kn source kafka create mykafkasrc \
-    --servers my-cluster-kafka-bootstrap.kafka.svc:9092 \
-    --topics my-topic --consumergroup my-consumer-group \
+$ kn source kafka create <kafka_source_name> \
```
Comment (mgencur) [inline modules/serverless-kafka-source-kn.adoc:30]: One of the "Example outputs" in this file still references `source: /apis/v1/namespaces/default/kafkasources/mykafkasrc#my-topic` . Perhaps it should be something like `source: /apis/v1/namespaces/default/kafkasources/example-kafka-source#example-topic` ?

**Review by mgencur — COMMENTED** [review]
Just one minor thing. Otherwise looks good.

File: modules/serverless-kafka-source-kn.adoc (hunk 5 above)
Comment (abrennan89) [reply modules/serverless-kafka-source-kn.adoc:30]: Good call, thanks!

**Comment by mgencur** [comment]
Thanks for the fix.
QE Approved.

File: modules/apiserversource-yaml.adoc [hunk 6]
```diff
@@ -115,15 +112,15 @@ spec:
...
+. Apply the `Service` YAML file:
 +
 [source,terminal]
 ----
-$ oc apply -f service.yaml
+$ oc apply -f <filename>
 ----
 
-. To create a trigger from the `default` broker that filters events to the service created in the previous step, create a file named `trigger.yaml` and copy the following sample code into it:
+. Create a `Trigger` object as a YAML file, that filters events from the `default` broker to the service created in the previous step:
```
Comment (ahardin-rh) [inline modules/apiserversource-yaml.adoc:123]: I think you can drop the comma after file now that the sentence is restructured and no longer has an introductory clause

File: modules/apiserversource-yaml.adoc [hunk 7]
```diff
@@ -140,22 +137,22 @@ spec:
...
+. Apply the `Trigger` YAML file:
 +
 [source,terminal]
 ----
-$ oc apply -f trigger.yaml
+$ oc apply -f <filename>
 ----
 
-. To create events, launch a Pod in the default namespace:
+. Create events by launching a pod in the default namespace. You can do this by entering the following command:
```
Comment (ahardin-rh) [inline modules/apiserversource-yaml.adoc:148]: Create events by launching a pod in the default namespace by entering the following command:

**Comment by ahardin-rh** [comment]
@abrennan89 Just a few minor suggestions from me. Looks good!

File: modules/apiserversource-yaml.adoc (hunk 7 above)
Comment (abrennan89) [reply modules/apiserversource-yaml.adoc:148]: Ended up just removing the enter the command bit til we have steady guidelines on it 🙂

**Pull request merged**